src/functions-and-arrays.js

```diff
@@ -1,20 +1,54 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(a, b) {
+  if (a>b) {
+    return a;
+  } else if (a<b) {
+    return b;
+  } else {
+    return a, b;
+  }
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(array) {
+  if (array.length === 0) {
+    return null;
+  }
+  let longestWord = '';
+  for (let i = 0; i < array.length; i++) {
+    
+    if (array[i].length > longestWord.length) {
+      longestWord = array[i];
+    }
+  }
+  return longestWord;
+  }
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(array) {
+  if (array.length === 0) {
+    return 0;
+  }
+  let sumOfNumbers = 0;
+  for (let i= 0; i < array.length; i++) {
+    
+    sumOfNumbers += array[i];
+    
+    if (array[i] === 0) {
+      return 0;
+    } else {
+      return sumOfNumbers;
```

Comment:
Here that return is inside the ` for...loop`  if you move it outside and remove the `if...else` that function would work correctly